docs/changelog-fragments.d/957.internal.md

```diff
@@ -0,0 +1,9 @@
+Added new subcommand *settings* to output all current ansible-navigator
```

Comment:
Also, this change doesn't seem to be contributor-facing. I think it's an end-user-facing feature meaning that this fragment file should be renamed from `internal` to something like `feature`.

The contribution docs at https://ansible-navigator.rtfd.io/en/latest/contributing/guidelines/#adding-change-notes-with-your-prs don't explicitly explain the fragment types but there's a hint there to look at https://github.com/ansible/ansible-navigator/blob/f4c0fdf/pyproject.toml#L13-L46 for the full list.

Check out this fragment preview in the docs build for the current PR: https://ansible-navigator--957.org.readthedocs.build/en/957/changelog/#contributor-facing-changes.

Reply:
We definitely need to update this.